obs-to-maven

```diff
@@ -40,40 +39,94 @@ def make_str(s):
         s = s.decode('utf-8')
     return s
 
+class Rpm:
```

Comment:
nitpick: Since this is rather a custom type than a class, a [namedtuple](https://docs.python.org/3/library/collections.html#collections.namedtuple) could be used here.